vos/vos/commands/vcp.py

```diff
@@ -55,6 +64,10 @@ class Nonlocal():
         exit_code = 0
 
     parser = CommonParser(description=DESCRIPTION)
+    parser.add_argument(
+        "--resource-id", default=None,
+        help="resource URI to use to lookup the storage inventory service "
+             "(Minoc)")
```

Comment:
I'm pretty sure users won't know what Minoc is.  Maybe we can just omit that?

Reply:
Removed but I think we need something for users to be able to identify services.